third_party/silabs/efr32_sdk.gni

```diff
@@ -204,11 +205,21 @@ template("efr32_sdk") {
       "${sl_ot_efr32_root}",
     ]
 
+    if (silabs_family == "efr32mg24") {
```

Comment:
Yes but it's not taking the included files for board 4186C we tried a lot of times, and if we separate it it works fine otherwise we are getting build error for this files if you want i can show you the issue

Reply:
Sounds odd. There is something else at play here.

Reply:
I'll raise JIRA on this to investigate further